Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rustreexo"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
Not sure if we should do this here. We probably need another release, but let's do it in another pr